app/src/main/java/com/team7/cmput301/android/theirisproject/LoginActivity.java

```diff
@@ -33,10 +39,28 @@ public void onClick(View view) {
             }
         });
 
+        /**
+         * loginButton's clickListener will take username input field and send intent
+         * to ProblemListActivity where it will search all problems under given intent value "user"
+         * */
         loginButton.setOnClickListener(new View.OnClickListener() {
-
             @Override
             public void onClick(View view) {
+                // Create login request, and start new activity if id is found
+                new LoginTask(new Callback<String>() {
+                    @Override
+                    public void onComplete(String res) {
+                        // act accordingly based on result "" if error, else success
+                        if(res != "") {
+                            // start new activity with given id from login request
+                            Intent intent = new Intent(LoginActivity.this, ProblemListActivity.class);
+                            intent.putExtra("user", res);
```

Comment:
The extra ID "user" should probably be made into a constant so it's maintainable and can be easily refactored